### PR TITLE
scripts: move gen_cyclic_test to CI repository 

### PR DESCRIPTION
**Pull request by bchedotel**
Move the document generation script for cyclictest
results to the CI directory, since this script is CI-specific.

### CODE REVIEW

File: scripts/gen_cyclic_test.sh [hunk 1]
```diff
@@ -47,14 +48,19 @@ parse_options()
...
+                ;;
+
             -h|--help)
                 print_usage
                 exit 1
                 shift
                 break
                 ;;
 
-            -|--)
```
Comment (eroussy) [inline scripts/gen_cyclic_test.sh:57]: Why this modification ?

File: scripts/gen_cyclic_test.sh [hunk 2]
```diff
@@ -81,17 +87,25 @@ plot_graph()
     local output_file=$3
     local histogram_file=$4
 
+    # Append the limit line if LIMIT is set
```
Comment (eroussy) [inline scripts/gen_cyclic_test.sh:90]: Wrong comment

**Review by dupremathieu — CHANGES_REQUESTED** [review]
Add a commit description.

**Review by dupremathieu — COMMENTED** [review]
If the limit in launch-yocto is always 100, why make the limit changeable here?

File: scripts/gen_cyclic_test.sh [hunk 3]
```diff
@@ -143,6 +158,31 @@ get_max_latency()
     echo $(grep "Max Latencies" "$input_file" | tr " " "\n" | sort -n | tail -1 | sed s/^0*//)
 }
 
+# Define template
```
Comment (eroussy) [inline scripts/gen_cyclic_test.sh:161]: It is better to put all the adoc template part in the CI repository.

May be you could move this entire script in the CI repository ?

**Review by eroussy — CHANGES_REQUESTED** [review]

**Review by eroussy — APPROVED** [review]

**Review by dupremathieu — APPROVED** [review]

**Pull request merged**